examples/openapi-tools/quickstart-mp/README.md

```diff
@@ -0,0 +1,258 @@
+# Helidon OpenAPI Generator example for Helidon MP server and client
+
+The goal of this example is to show how a user can easily create a Helidon MP server or client from an OpenAPI document using OpenAPI Generator.  
+
+Here we will show the steps that a user has to do to create Helidon MP server and client using OpenAPI Generator and what has to be done to make the generated server and client fully functional.
+
+For generation of our projects we will use `openapi-generator-cli.jar` that can be downloaded form the maven repository (instructions and other options can be found [here](https://openapi-generator.tech/docs/installation)) and OpenAPI document `quicksrart.yaml` that can be found next to this `README.md`.
+
+## Build, prepare and run the Helidon MP server
+
+To generate Helidon MP server at first we create `mp-server` folder and then inside it we run the following command where `path-to-generator` is the directory where you downloaded the generator CLI JAR file and `path-to-openapi-doc` is the folder where `quickstart.yaml` is located:
+```bash
+java -jar path-to-generator/openapi-generator-cli.jar \
+          generate \
+          -g java-helidon-server \  
+          --library mp \
+          -i path-to-openapi-doc/quickstart.yaml
+```
+
+When this command finishes its work in the folder `mp-server` we will find the generated project where the most interesting parts are located inside `api` and `model` packages.
+The package `api` contains interfaces that represent endpoints for our server and implementations with stubs for them. 
+These implementations we need to change to implement our business logic.
+The package `model` contains classes that represent transport objects that will be used by our endpoints to receive requests and sent responses.
+
+Let's change a little class `MessageServiceImpl` for our example :
+1) Add annotation `@ApplicationScoped` to this class.
+2) Add field that will contain default message for our endpoints :
+```java
+    private final AtomicReference<Message> defaultMessage = new AtomicReference<>();
+```
+3) Add default constructor to the class :
+```java
+    public MessageServiceImpl() {
+        Message message = new Message();
+        message.setMessage("World");
+        message.setGreeting("Hello");
+        defaultMessage.set(message);
+    }
```

Comment:
The described scenario works and I do not need to change anything because I used `@ApplicationScoped` for `MessageServiceImpl` class, but I have noticed that the method `public Message getMessage(@PathParam("name") String name)` does not work as expected so I am going to change it.

Reply:
Oh, yes. I missed that step in the instructions.

I did not notice a problem with the `getMessage` method but I look forward to seeing your changes.